packages/jaeger-ui/src/model/ddg/DdgEVManager.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { compareVisibilityKeys } from './visibility-key';
+
+import {
+  PathElem,
+  DdgEdge,
+  DdgVertex,
+  TDdgModel,
+  TDdgPathElemsByDistance,
+  TDdgPath,
+  TDdgServiceMap,
+} from './types';
+
+export default class DdgEVManager {
+  lastVisibilityKey: string;
+  pathElemsByDistance: TDdgPathElemsByDistance;
+  pathElemToVertex: Map<PathElem, DdgVertex>;
+  paths: TDdgPath[];
```

Comment:
This is unused, should be removed until a use case arises for needing paths (which probably won't be in this component)